src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import core.basesyntax.suppliers.FigureSupplier;
+
+public class Main {
+    public static final int FIGURES_AMOUNT = 5;
+
+    public static void main(String[] args) {
```

Comment:
Please read requirements carefully and fix this method

Reply:
not fixed, half of the figures should be random, and the other half should be default, check the task description for details, or feel free to ask questions